NSS-Day-Cohort-42/nutshell-lizard-collectors
# Cj lizard style

# Steps to test
1. `git fetch --all`
1. `git checkout cj-lizard-style`
1. run json-server `json-server -p 8088 -w database.json`
1. serve the page and you should see a logo and css changes primarily in the news section, and friends list
